Sources/BraveWallet/Crypto/Stores/BuyTokenStore.swift

```diff
@@ -22,9 +22,31 @@ public class BuyTokenStore: ObservableObject {
   @Published var selectedCurrency: BraveWallet.OnRampCurrency = .init()
   
   /// A map of list of available tokens to a certain on ramp provider
-  var buyTokens: [BraveWallet.OnRampProvider: [BraveWallet.BlockchainToken]] = [.ramp: [], .sardine: [], .transak: []]
+  var buyTokens: [BraveWallet.OnRampProvider: [BraveWallet.BlockchainToken]]
   /// A list of all available tokens for all providers
   var allTokens: [BraveWallet.BlockchainToken] = []
+  
+  /// The supported `OnRampProvider`s for the currently selected currency and device locale.
```

Comment:
will need to update the comment here with the device locale.

Reply:
We get the region from the device _locale_: https://github.com/brave/brave-ios/pull/7879/files/07b7bcbc1aaa9a697ecca7e36a91fbbfe33f4c00#diff-29b66424de47bfdd2a8a12b7664a302773cd703aea0a2ba33946715e23342e0bR363